projects/schema-form/package.json

```diff
@@ -24,15 +24,17 @@
     "Juan Manuel Verges",
     "Daniele Pecora <daniele.pecora@googlemail.com>"
   ],
+  "dependencies": {
```

Comment:
@andrea11 using the `dependencies` entry for a library is not recommended (as it might result in different versions of the same package within the end-user project.

Reply:
Hi, ebrehault I looked on internet. I think that you are right. It would be better to keep in peerDependencies. Thanks